topics/dev/tutorials/bioblend-api/slides.html

```diff
@@ -287,6 +287,26 @@
 
 ---
 
+## Galaxy API Modernization
+
+.left[Moving to [FastAPI](https://fastapi.tiangolo.com/)]
+
+- Main advantages:
+  - [Async requests](https://fastapi.tiangolo.com/async/)
+  - Subscriptions via [websockets](https://fastapi.tiangolo.com/advanced/websockets/)
+  - [OpenAPI](https://spec.openapis.org/oas/v3.1.0) interactive documentation
+  - Reduce maintenance burden
+- Challenges:
+  - Need to refactor over 200 endpoints
+
+???
+
+- The Galaxy API is currently in the process of migration towards FastAPI, which is a modern framework for building REST APIs in Python with really interesting features.
+- Some of the main advantages are the ability to take work out of the request cycle by making asynchronous requests or avoid inefficient polling by using WebSockets subscriptions which translates into improved performance.
```

Comment:
```suggestion
- For example, FastAPI can serve requests with better performances using asynchronous coroutines.
- Another performance advantage is the ability to avoid inefficient polling by using WebSockets subscriptions.
```